### ISSUE TITLE
`[caprieval]` with ensemble as reference

### ISSUE THREAD

**Issue by VGPReys** [haddocking/haddock3#1132]
When providing an ensemble as in the parameter `reference_fname`, no computation is performed, as there is a bug in the alignment procedure.

## **Module/Workflow/Library affected**

`[caprieval]`

## **Actual behavior**

part of the `log`
```
Running [caprieval] module
[2024-11-05 16:40:36,091 __init__ INFO] [caprieval] Found previous CNS step: 1_rigidbody
[2024-11-05 16:40:36,192 __init__ INFO] [caprieval] Saved scoring weights to: weights_params.json
[2024-11-05 16:41:17,429 libutil INFO] Selected 30 cores to process 1782 jobs, with 32 maximum available cores.
[2024-11-05 16:41:17,431 libparallel INFO] Using 30 cores
[2024-11-05 16:41:17,468 capri WARNING] Alignment failed between ../data/4_caprieval/reference.pdb and [pdb|2024-11-05 15:50:45] /hddb/roxana/protein-RNA_tutorial/7q4l_redock_bound/3_seletopclusts/3_seletopclusts/cluster_23_model_3.pdb, skipping...
[2024-11-05 16:41:17,473 capri WARNING] Alignment failed between ../data/4_caprieval/reference.pdb and [pdb|2024-11-05 15:50:46] /hddb/roxana/protein-RNA_tutorial/7q4l_redock_bound/3_seletopclusts/3_seletopclusts/cluster_38_model_3.pdb, skipping...
```

## **Steps to reproduce the behavior**

```toml
[caprieval]
reference_fname = "ensemble.pdb"
```

## **Suggestions on how to fix it**

- Parse the `reference_fname`
- Select first model
- Make comparison against all models ?

## **Additional context**

Discovered by our best user: Roxana !

**Comment by mgiulini** [comment]
I would probably go with selecting the first model within the ensemble

**Comment by VGPReys** [comment]
Yes, me too, much simpler.

But there is an interesting situation where you would like to compare your result to a NMR structure represented by an ensemble of conformation, and where there is no trivial ways to assess the quality of your results.

**Comment by VGPReys** [comment]
We could create a new parameter to tackle this purpose.

**Comment by mgiulini** [comment]
yes, but handling that case would make things very complicated code-wise (multiple capri files to be written etc), while a much simpler solution is to run caprieval sequentially over all the elements of the ensemble.
we just have to put a warning when an ensemble is detected, maybe something along these lines:
`WARNING: selecting first model out of reference ensemble: consider providing different ensemble elements as reference_fname in sequential caprieval modules`

**Comment by VGPReys** [comment]
Closed by #1133
